Source/Dockerfile

```
@@ -0,0 +1,6 @@
+FROM nginx:1.17.6
+RUN apt-get update && apt-get install -y nginx-extras
```

In order to get the final image smaller:
```suggestion
RUN apt-get update && apt-get install -y nginx-extras && rm -rf /var/lib/apt/lists/*
```

---

Will do that in next PR!